contrib/clojure-package/examples/infer/predictor/src/infer/predictor_example.clj

```diff
@@ -0,0 +1,105 @@
+;; Licensed to the Apache Software Foundation (ASF) under one or more
+;; contributor license agreements.  See the NOTICE file distributed with
+;; this work for additional information regarding copyright ownership.
+;; The ASF licenses this file to You under the Apache License, Version 2.0
+;; (the "License"); you may not use this file except in compliance with
+;; the License.  You may obtain a copy of the License at
+;;
+;;    http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+
+(ns infer.predictor-example
+  (:require [org.apache.clojure-mxnet.context :as context]
+            [org.apache.clojure-mxnet.dtype :as dtype]
+            [org.apache.clojure-mxnet.image :as image]
+            [org.apache.clojure-mxnet.infer :as infer]
+            [org.apache.clojure-mxnet.io :as mx-io]
+            [org.apache.clojure-mxnet.layout :as layout]
+            [org.apache.clojure-mxnet.ndarray :as ndarray]
+            [org.apache.clojure-mxnet.shape :as shape]
+            [clojure.java.io :as io]
+            [clojure.string :refer [join split]]
+            [clojure.tools.cli :refer [parse-opts]])
+  (:gen-class))
+
+(defn check-valid-file
+  "Check that the file exists"
+  [input-file]
+  (.exists (io/file input-file)))
+
+(def cli-options
+  [["-m" "--model-path-prefix PREFIX" "Model path prefix"
+    :default "models/resnet-152/resnet-152"
```

Comment:
can we make the default resnet-18 because that is what the README says to download?

Reply:
done